client/domain/FeedbackObjectFactory.js

```diff
@@ -22,6 +22,10 @@ tie.factory('FeedbackObjectFactory', [
     var Feedback = function(answerIsCorrect) {
       this._paragraphs = [];
       this._answerIsCorrect = answerIsCorrect;
+      // This records what message was displayed with this feedback.
+      // If no message (or no message with additional potential messages)
+      // was displayed, this will remain at -1.
```

Comment:
Although I don't actually do any work with it as an int, so I can change it to null if you'd prefer -- I usually just use -1 here because the name suggests an int, rather than an object to me.